geosys/bridge_api_wrapper.py

```diff
@@ -438,15 +439,17 @@ def get_samz_map(
         # Construct map creation parameters
         request_data = {
             "SeasonField": {
-                "Id": season_field_id
+                "Id": None,
```

Comment:
Is the season field id always None?

Reply:
Based on the response, they all had a value of `None` or `Null` value so i opted not to use it and instead use geometry.